package.json

```diff
@@ -15,9 +15,9 @@
     "build": "yarn build-sass && yarn build-js && yarn copy-assets",
     "build-sass": "rm -rf dist/public/style.css && sass --no-source-map src/assets/scss/application.scss dist/public/stylesheets/application.css --style compressed",
     "copy-assets": "mkdir -p dist && copyfiles -u 1 src/**/*.njk dist/ src/locales/**/** dist/ && copyfiles -u 3 src/assets/javascripts/*.js dist/public/scripts && copyfiles -u 3 src/assets/images/* dist/public/images",
-    "build-js": "yarn build-js:application && yarn build-js:cookies && yarn build-js:all",
-    "build-js:application": "mkdir -p dist/public/javascripts; uglifyjs src/assets/javascripts/application.js  node_modules/di-ipv-cri-common-express/src/assets/javascript/application.js --beautify -o dist/public/javascripts/application.js",
-    "build-js:cookies": "mkdir -p dist/public/javascripts; uglifyjs node_modules/di-ipv-cri-common-express/src/assets/javascript/cookies.js --beautify -o dist/public/javascripts/cookies.js",
+    "build-js": "yarn build-js:application && yarn build-js:analytics && yarn build-js:all",
+    "build-js:application": "mkdir -p dist/public/javascripts; uglifyjs src/assets/javascripts/application.js --beautify -o dist/public/javascripts/application.js",
+    "build-js:analytics": "mkdir -p dist/public/javascripts; uglifyjs src/assets/javascript/analytics/**/*.js src/assets/javascript/analytics/*.js node_modules/di-ipv-cri-common-express/src/assets/javascript/cookies.js --beautify -o dist/public/javascripts/analytics.js",
     "build-js:all": "mkdir -p dist/public/javascripts; uglifyjs node_modules/govuk-frontend/govuk/all.js node_modules/hmpo-components/all.js --beautify -o dist/public/javascripts/all.js",
     "minfiy-build-js": "uglifyjs src/assets/javascript/application.js -o src/assets/javascript/application.js -c -m && uglifyjs src/assets/javascript/cookies.js -o src/assets/javascript/cookies.js -c -m",
```

Comment:
Do we want to update this to include `analytics.js`?

Reply:
Sorry @Caitlin-cooling not sure what you mean here - we don't have an analytics.js

Reply:
Ignore me